src/pages/Home/components/Days/Days.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import {Card} from "./Card";
+import {Tabs} from "./Tabs";
+import {DaysStyles} from "./DaysStyles";
+
+interface Props{
+}
+export interface Day{
+  day: string,
+  day_info: string,
```

Comment:
Call it with Uppercase
```suggestion
  dayInfo: string,
```